hw/hw_11_task_01.py

```diff
@@ -0,0 +1,39 @@
+"""
+Vasya implemented nonoptimal Enum classes.
+
+Remove duplications in variables declarations using metaclasses.
+from enum import Enum
+class ColorsEnum(Enum):
+    RED = "RED"
+    BLUE = "BLUE"
+    ORANGE = "ORANGE"
+    BLACK = "BLACK"
+class SizesEnum(Enum):
+    XL = "XL"
+    L = "L"
+    M = "M"
+    S = "S"
+    XS = "XS"
+Should become:
+class ColorsEnum(metaclass=SimplifiedEnum):
+    __keys = ("RED", "BLUE", "ORANGE", "BLACK")
+class SizesEnum(metaclass=SimplifiedEnum):
+    __keys = ("XL", "L", "M", "S", "XS")
+assert ColorsEnum.RED == "RED"
+assert SizesEnum.XL == "XL"
+"""
+
+
+class SimplifiedEnum(type):
+    def __new__(cls, name, bases, dct):  # noqa
+        cls_instance = super().__new__(cls, name, bases, dct)
+        cls_instance.__values__ = dct[f"_{name}__keys"]
+        for key in cls_instance.__values__:
+            setattr(cls_instance, key, key)
+        return cls_instance
+
+    def __iter__(cls):
```

Comment:
Is it required?

Reply:
I thought we must create a class that acts like real enum.